rrule/tests/rrule.rs

```diff
@@ -4795,3 +4796,46 @@ fn test_between_inclusive_both_hit() {
         rrule.all_between(before, after, inc).unwrap()
     );
 }
+
+#[test]
+fn rrule_to_and_from_str() {
```

Comment:
Consider adding this to a new test file called `serde` (same file as serde tests for `RRuleProperties` should be fine also IMO) or something similar. At the moment the tests in this library is pretty much spread around and not logically grouped, I hope we can improve on this in the future!